src/LuauAST/impl/create.ts

```diff
@@ -90,7 +90,7 @@ export function id(name: string) {
  * @param text The text of the comment
  */
 export function comment(text: string) {
-	return luau.create(luau.SyntaxKind.Comment, { text: " " + text });
+	return luau.create(luau.SyntaxKind.Comment, { text });
```

Comment:
This shorthand is meant for making simple single line comments (for things like macros, etc.). I don't think you should change this.

Instead, use `luau.create(luau.SyntaxKind.Comment, { text: "" })` where needed.

Reply:
It's taken care of in the comment renderer. It causes double indentation for block comments.